solutions/client-landing-zone/client-folder/firewall-policy/policy.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# The following rules are automatically created when the firewall policy is created:
+# 2147483644 default egress rule ipv6 Egress IPv6 ranges: ::/0 all Goto next
+# 2147483645 default ingress rule ipv6 Ingress IPv6 ranges: ::/0 all Goto next
+# 2147483646 default egress rule Egress IPv4 ranges: 0.0.0.0/0 all Goto next
+# 2147483647 default ingress rule Ingress IPv4 ranges: 0.0.0.0/0 all Goto next
+#########
+# Client Compute Firewall Policy on folder client-folder
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeFirewallPolicy
+metadata:
+  name: client-name-client-folder-fwpol # kpt-set: ${client-name}-client-folder-fwpol
+  namespace: client-name-networking # kpt-set: ${client-name}-networking
+spec:
```

Comment:
missing depends-on

Reply:
can't seem to add a depends-on for a folder that was created in another root-sync
leaving out for now, we'll have to investigate